tests/reals_ut/azure_c_util_reals_ut.c

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "testrunnerswitcher.h"
+
+#define REGISTER_GLOBAL_MOCK_HOOK(original, real) \
+    (original == real) ? (void)0 : (void)1;
+
+#include "../tests/reals/real_constbuffer.h"
```

Comment:
>include [](start = 1, length = 7)

real_crt_abstractions and real_doublylinkedlist missing?

Reply:
Added both of them. And the new test actually caught missing things in the renames!

---
In reply to: [369869348](https://github.com/Azure/azure-c-util/pull/12#discussion_r369869348) [](ancestors = 369869348)